main/modes/picross/picross_menu.c

```diff
@@ -373,7 +373,15 @@ void picrossButtonCb(buttonEvt_t* evt)
             //Pass button events from the Swadge mode to the menu
             if (evt->down)
             {
-                meleeMenuButton(pm->menu, evt->button);
+                if (pm->screen == PICROSS_OPTIONS && evt->button == BTN_B)
```

Comment:
Same comment as fighter

Reply:
I have no thoughts here, no clue what it should be - you tell me. I copied flying donut when I was just getting started originally and had no idea how any of it was configured. (Now I know what's going on, but not about to refactor). Whatever changes are appropriate should get copied here too I guess.

Reply:
I don't think there could be any issues with an unhandled menu option here, since this should only try to call the menu callback when the options menu is shown. Also since both menu callbacks are combined, it looks like it would be handled either way.